### DISCUSSION TITLE
/giveall

### DISCUSSION THREAD

**Discussion by danechek** [EssentialsX/Essentials#4063] [Ideas and Feature Suggestions] [answered]
Add these commands, please:
/giveall <item> <meta> - give item to all players
/giveall hand - give copy of item from your hand to all players

**Comment by Pheotis** [comment, marked as answer]
It is currently possible to give all online players an item with `/give * <itemname>`.
`give *` can be assigned the alias `giveall`.

It also possible to get the name of what is in your hand with `/itemdb`.
As possible tweak to that cmd, perhaps the returned `Item:` string could be made copyable?

![image](https://user-images.githubusercontent.com/49039536/111577917-82d1f800-8789-11eb-8748-09bcfa1c15ff.png)

**Reply by danechek** [reply]
Under the "item from hand" I meant: For example, I gave myself a custom potion with certain parameters or effects and then issued it to all players. Or via /give * can also be any item with any parameters to give everyone at once?

**Reply by Pheotis** [reply]
It is possible to `/give *` everyone an item, complete with parameters; for more information, see [this page](http://wiki.ess3.net/wiki/Item_Meta).

For example:
`/give * potion 1 name:&d&lCustom_Potion lore:&7Fancy,_eh? effect:luck power:5 duration:86400`
![image](https://user-images.githubusercontent.com/49039536/111727663-b40dff00-8841-11eb-8578-5b19bc1e4f43.png)



That said, item meta is not reflected by the `/itemdb` command... so that could probably be tweaked as well.
![image](https://user-images.githubusercontent.com/49039536/111727894-24b51b80-8842-11eb-824d-88c3e72d4b12.png)
